### PR TITLE
docker

### PR DESCRIPTION
**Pull request by tannernelson**

### CODE REVIEW

**Comment by tannernelson** [comment]
Moving #7 here.

File: vapor [hunk 1]
```diff
@@ -699,6 +699,34 @@ extension Docker {
         static let id = "init"
 
         static func execute(with args: [String], in directory: String) {
+            let quiet = args.contains("--verbose") ? "" : "-s"
+
+            do {
+                print("Downloading Dockerfile...")
+                try run("curl -L \(quiet) docker.qutheory.io -o Dockerfile")
+            } catch {
+                print("Could not download Dockerfile.")
```
Comment (feinstruktur) [inline vapor]: Should this be `fail(…)`?

Comment (tannernelson) [inline vapor]: Good catch.

File: vapor [hunk 2]
```diff
@@ -699,6 +699,34 @@ extension Docker {
         static let id = "init"
 
         static func execute(with args: [String], in directory: String) {
+            let quiet = args.contains("--verbose") ? "" : "-s"
+
+            do {
+                print("Downloading Dockerfile...")
+                try run("curl -L \(quiet) docker.qutheory.io -o Dockerfile")
```
Comment (feinstruktur) [inline vapor]: This just nuked my local `Dockerfile` :) I’ll add a check.

Comment (tannernelson) [inline vapor]: D'oh >_<

**Comment by feinstruktur** [comment]
Right, I think that’s it for this round of changes :)

One other thing I noticed is that inside the container, we build and run like this:
```
swift build && .build/debug/App
```
So it’s always a debug build, which is fair enough, because you wouldn’t deploy with this mechanism of volume mapping anyway.

This is note for future discussion/extension that we could make `vapor` available inside the container and pass through args from the outside. This is a bit like Inception for containers. May be a tad confusing, just thought I’d throw it out there.

**Comment by tannernelson** [comment]
> vapor available inside the container and pass through args from the outside

Could be cool. We can look into this in the future.

> So it’s always a debug build

That seems fine, especially for now since there is a bug in `05-09` which breaks compiling release.

**Comment by tannernelson** [comment]
`docker.qutheory.io` is setup.

**Pull request merged**